packages/Notification/__tests__/Notification.spec.jsx

```diff
@@ -58,12 +51,6 @@ describe('<Notification />', () => {
   })
 
   describe('error variant', () => {
-    it('bolds the content', () => {
```

Comment:
Do we want to test for default `font-weight`?

Reply:
Done